### PR TITLE
[Neo CLI Optimization] fix exception message

### PR DESCRIPTION
**Pull request by Jim8y**
# Description
PR https://github.com/neo-project/neo/commit/38cc0e9384a31f3b492225a190c38aba72d7c7f9 fiexd the dll missing problem on windows server, but its checking and error message is too simple and shows installing exe on macos.

Fixes # https://github.com/neo-project/neo/issues/3199

## Type of change

- [x] Optimization (the change is only an optimization)
- [ ] Style (the change is only a code style for better maintenance or standard purpose)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Neo.CLI/CLI/MainService.cs [hunk 1]
```diff
@@ -377,15 +378,46 @@ public async void Start(CommandLineOptions options)
...
+                    }
+
+                }
+                else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    ConsoleHelper.Error("DLL not found, please get libleveldb.dylib." +
+                                        Environment.NewLine +
+                                        "For AppleSilicon https://github.com/neo-project/neo/releases/download/v3.7.4/neo-cli-osx-arm64.zip" +
+                                        Environment.NewLine +
+                                        "For Intel https://github.com/neo-project/neo/releases/download/v3.7.4/neo-cli-osx-x64.zip" +
```
Comment (cschuchardt88) [inline src/Neo.CLI/CLI/MainService.cs:410]: Just send them to the release page. It much simpler to maintain.

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]

File: src/Neo.CLI/CLI/MainService.cs (hunk 1 above)
Comment (shargon) [reply src/Neo.CLI/CLI/MainService.cs:410]: Agree

File: src/Neo.CLI/CLI/MainService.cs [hunk 2]
```diff
@@ -377,15 +378,44 @@ public async void Start(CommandLineOptions options)
...
+                                        Environment.NewLine +
+                                        "Press any key to exit.");
+                }
+
                 Console.ReadKey();
                 Environment.Exit(-1);
+            }
+            catch (DllNotFoundException)
+            {
+
```
Comment (roman-khimov) [inline src/Neo.CLI/CLI/MainService.cs:418]: Just ignoring it here?

Comment (Jim8y) [reply src/Neo.CLI/CLI/MainService.cs:399]: fixed

**Review by cschuchardt88 — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**